uaclient/security.py

```diff
@@ -256,31 +255,19 @@ def __init__(self, client: UASecurityClient, response: "Dict[str, Any]"):
         self.response = response
         self.client = client
 
+    def __eq__(self, other) -> bool:
```

Comment:
I am missing why we need this override here and on the USN class as well

Reply:
> I am missing why we need this override here and on the USN class as well

Ahh sorry, I should have noted that, it is really only for ease of unittesting. in some of our pytest.mark.parametrize setup it allows us to instead create USN() CVE() in the list values and [do a direct equality comparison here](https://github.com/canonical/ubuntu-advantage-client/pull/1492/files#diff-1510e0bbe0ff95e8aa5c9a0d886d046cb9d1823c3eaf3bc6295cce6596779bceR304). This was we don't have to assert <expected_usn>.response  == <actual_usn>.response etc.